src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs

```diff
@@ -0,0 +1,1115 @@
+namespace System.Runtime.Intrinsics.X86
```

Comment:
Needs a license header like the other ref files, e.g.:

https://github.com/dotnet/corefx/blob/fd3004d31e233ff42cab77f1ef0a49ea388bc9ac/src/System.Runtime/ref/System.Runtime.cs#L1-L6